packages/itwinui-react/src/core/Tile/index.ts

```diff
@@ -5,3 +5,4 @@
 export { Tile } from './Tile.js';
 export type { TileProps } from './Tile.js';
 export default './Tile';
+export * from './TileNew';
```

Comment:
Changing this to `'./TileNew.js'` should fix the `yarn lint` issue in the PR

Reply:
Nice! Thanks for the tip. I updated this.